Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,24 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
+  const result = tasks
+    .map(item => item.duration / 60)
```

Comment:
I think I used `item` as the name of a variable when doing `map/filter/reduce` during the lesson as well. Nevertheless, I think the name `task` would be more clear in this case!